python/tests/test_transforms.py

```diff
@@ -422,6 +422,7 @@ def test_void_transform() -> None:
 
 class TestType(IcebergBaseModel):
     __root__: Transform[Any, Any]
+    __test__ = False
```

Comment:
Was this fixed in the other PR?

Reply:
I think that was a PR to the 0.3.0 branch, so we can create another RC from that branch without having the latest pyarrow changes released along with it.